### PR TITLE
Replace EE License Placeholder text with filename

### LINKED ISSUE #5528: COPYING.txt contains placeholder text
Line 7:

> &lt;insert file name of the MEE license&gt;

Should be replaced with the file name of the Magento2 Enterprise Edition License.

### PR DESCRIPTION
**Pull request by navarr**
Resolves #5528

### CODE REVIEW

File: COPYING.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
 Please see LICENSE.txt for the full text of the OSL 3.0 license or contact license@magentocommerce.com for a copy.
 
 Subject to Licensee's payment of fees and compliance with the terms and conditions of the MEE License, the MEE License supersedes the OSL 3.0 license for each source file.
-Please see <insert file name of the MEE license> for the full text of the MEE License or visit http://magento.com/legal/terms/enterprise.
\ No newline at end of file
+Please see LICENSE_EE.txt for the full text of the MEE License or visit http://magento.com/legal/terms/enterprise.
```
Comment (orlangur) [inline COPYING.txt]: I don't think referring to a file which is actually missing in CE repository is correct.

Is there any problem with adding this file to CE repository also? It does not seem to me to bring any confusion as each module [defines](https://github.com/magento/magento2/blob/develop/app/code/Magento/Backend/composer.json#L26) applicable licenses.

Comment (orlangur) [inline COPYING.txt]: I don't think referring to a file which is actually missing in CE repository is correct.

Is there any problem with adding this file to CE repository also? It does not seem to me to bring any confusion as each module [defines](https://github.com/magento/magento2/blob/develop/app/code/Magento/Backend/composer.json#L26) applicable licenses.

**Review by orlangur — CHANGES_REQUESTED** [review]

**Comment by navarr** [comment]
I am personally uncomfortable adding the LICENSE which as of now is only available in software that is supposed to be kept private to the repository.

I would not want to open myself nor my employer up to any sort of legal issue related to revealing things we shouldn't.

File: COPYING.txt (hunk 1 above)
Comment (navarr) [reply COPYING.txt]: I am personally uncomfortable adding the LICENSE which as of now is only available in software that is supposed to be kept private to the repository.

I would not want to open myself nor my employer up to any sort of legal issue related to revealing things we shouldn't.

**Comment by woosley-mg** [comment]
@navarr Wanted to comment on this directly, since I strongly support this change.  Because it will introduce another diff bomb (every file reporting a change), I'd prefer to defer this until the end of the year, when we will likely package it as a separate release.  Great work, and looking forward to never having to update this again!

**Comment by orlangur** [comment]
@vrann do you mean some response in internal discussion? Here we finished our conversation with PR author as he is really not in charge of adding new license file to repo :)

**Comment by vrann** [comment]
@orlangur exactly.

**Comment by vrann** [comment]
@navarr can you please merge with the latest develop? I'm ready to accept it

**Comment by magento-team** [comment]
@navarr thank you for your contribution to Magento 2 project

**Pull request merged**